src/infinitecanonicalmps.jl

```diff
@@ -249,21 +249,35 @@ function InfMPS(s::CelledVector, f::Function)
   return ψ = InfiniteCanonicalMPS(ψL, ψC, ψR)
 end
 
-function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange)
+function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange; ortho::String="right")
```

Comment:
I'm guessing this is a bit of a convention thing, but setting options like this via `Symbol`s ensures that the compiler can see this. In particular, it can then typically constant propagate this entire function and eliminate the branching.
Since it's in a kwarg, you can always consider "standardizing" the input beforehand to allow both?
```suggestion
function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange; ortho=:right)
    _ortho = ortho isa Symbol ? ortho : Symbol(ortho)
```

(of course, probably does not matter all that much)

Reply:
(But also as a convention we generally use strings for keyword arguments that select different options unless there is a performance reason, since it looks friendlier to users, but as you say we can just allow generic inputs and canonicalize the input.)